myria/test/test_query.py

```diff
@@ -190,16 +196,17 @@ def test_submit_plan(self):
 
     def test_submit_program(self):
         with HTTMock(local_mock):
-            program = 'COMPLETE_IMMEDIATELY'
+            program = 'CI = empty(i:int);\nstore(CI, CI);'
```

Comment:
I don't remember :(
I think we might be fine without  it -- I merged the tests  file.